doc/contributing/contributing_guidelines.md

```diff
@@ -188,7 +189,66 @@ Work in progress.
 [Top](#top)
 
 ##<a name="section9"></a>Documentation
-Work in progress.
+###<a name="section9.1"></a>Repository documentation
+There MUST exist a `doc/` folder at the root of the main repository. Within this folder, there will be a subfolder for each agent type, in addition to a subfolder for `cygnus-common`. For instance:
+
+* `doc/cygnus-ngsi`
+* `doc/cygnus-twitter`
+* `doc/cygnus-common`
+
+All the agent teams MUST provide any documentation aimed to be exposed to the Community within its documentation subfolder.
+
+In addition, any agent documentation subfolder MUST have at least the following elements:
+
+* `doc/<agent name>/installation_and_administration_guide/` → Any document regarding how to install and administrate the agent MUST be placed here. Markdown MUST be used for the documents within this subfolder.
+* `doc/<agent name>/user_and_programmer_guide/` → Any document regarding how to use and programme the agent MUST be placed here. Markdown MUST be used for the documents within this subfolder.
+
+The following elements SHOULD be present as well:
+
+* `doc/<agent name>/flume_extensions_catalogue/` → For each class added to the native Apache Flume library, there SHOULD be a document placed here. Markdown MUST be used for the documents within this subfolder.
+* `doc/<agent name>/quick_start_guide.md/` → Simple and ready-to-use commands/actions to be taken in order to quickly test the agent SHOULD be documented here. Markdown MUST be used.
+
+[Top](#top)
+
+###<a name="section9.1"></a>`readthedocs.org` documentation
+The documentation within the `doc/` folder MUST be exposed in the Interned by means of `readthedocs.org` service. In order to achieve this, a `mkdocs.yml` file MUST live in the root of the main repository acting as a hook.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/949/commits/540563c64137c0bf10bcebaa9cf8f4dfe26093e1